strongbox-commons/src/main/resources/application.properties

```diff
@@ -25,6 +27,9 @@ ehcache.disk.store.dir=strongbox/cache
 
 spring.servlet.multipart.enabled=false
 
+info.strongbox.version=${strongbox.version}
+info.strongbox.revision=${strongbox.revision}
```

Comment:
What's the purpose of these `info.*` variables? Don't we already have them a few lines above?

Reply:
It is necessary for the info actuator endpoint, isn't it? Without this properties, it does not work.

Reply:
Can't the actuator just use `strongbox.version` and `strongbox.revision`? I mean -- why do you need to have two sets of properties?

Reply:
Without that there is an exception and it does not work. I think that is necessary to leave it like that.

Reply:
I'm sorry, but could you please investigate this? We can't have two sets of properties. This will confuse people. We should only have the `strongbox.version` and `strongbox.revision` ones.

@sbespalov, @fuss86, @steve-todorov : Am I missing something here?

Reply:
From what I've seen in the documentation, this is necessary because the `info` controller displays everything from the `info.*` properties. However, it should also be possible to create a custom `InfoContributor` component which returns the information from the `${strongbox.version}`.

Reply:
Sounds good!

Reply:
I will try to implement it today and also test it.

Reply:
Looks good, thanks! :)